### ISSUE TITLE
Rubberband guide not accurate

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#1178]
Set some alternate displayed regions

```
MODEL.views[0].setDisplayedRegions([
{ assemblyName: 'volvox', refName: 'ctgA', start: 5000, end: 10000 },
{ assemblyName: 'volvox', refName: 'ctgA', start: 30000, end: 35000 },
])
```

The coordinate displayed in the popup is not the right bp in either column, should probably use a full pxToBp or something like this. Could display refName too (only when multiple refs are used or always, not sure)

![t1](https://user-images.githubusercontent.com/6511937/92184348-eebcc680-ee1e-11ea-8178-3052bad159f1.png)

**Comment by cmdcolin** [comment]
In case it's not clear, the "guide" is the mouseover thing when hovering the scale bar

**Comment by cmdcolin** [comment]
@teresam856 this might be a good issue for you also while looking at the other rubberband bar related things

**Comment by teresam856** [comment]
@cmdcolin I can take a look.  Are the calculations of the x axis not accurate?

**Comment by cmdcolin** [comment]
Probably add refName to the guide also

**Comment by teresam856** [comment]
Guides are now displaying accurate BP and popovers contain RefNames

- Vertical Guides

<img width="1440" alt="RubberBandGuide1" src="https://user-images.githubusercontent.com/45598764/93514640-4b6ab780-f8dc-11ea-8add-fd5dff458165.png">
<img width="1437" alt="RubberBandGuide2" src="https://user-images.githubusercontent.com/45598764/93514648-4dcd1180-f8dc-11ea-8c29-7688d06f12fd.png">
<img width="1434" alt="RubberBandGuideFlipped" src="https://user-images.githubusercontent.com/45598764/93514656-502f6b80-f8dc-11ea-8d4a-1964335b8518.png">

- Mouse Dragging Guides

<img width="1438" alt="RubberBandMouseDraggingGuideRegionFlipped" src="https://user-images.githubusercontent.com/45598764/93514473-1b231900-f8dc-11ea-9c76-0fa418b89f5b.png">
<img width="1426" alt="RubberBandMouseDraggingGuide" src="https://user-images.githubusercontent.com/45598764/93514486-1eb6a000-f8dc-11ea-8b96-c721d5e44ea6.png">
<img width="1438" alt="MouseDraggingGuide3" src="https://user-images.githubusercontent.com/45598764/93514894-95ec3400-f8dc-11ea-8161-4c1d23ca0d9c.png">

**Comment by teresam856** [comment]
This approach will continue to allow users to see guides past the displayed regions, but will accurately display the bp and refname based on the ruler above the Rubber band. 
This will also continue to allow user do mouse-dragging and zooming into regions that include areas outside the displayed regions.

**Comment by cmdcolin** [comment]
I think that looks accurate to me. I know the overview rubberband does not currently display refname in the guide either, could be one more small add on to this

**Comment by teresam856** [comment]
Yeah, I can add that to the overview rubberband as well.

**Comment by teresam856** [comment]
Adding RefNames to the Overview RubberBand Guides
<img width="1434" alt="OverviewRubberBandGuide" src="https://user-images.githubusercontent.com/45598764/93534838-b1663780-f8fa-11ea-9335-44b83d7d0b96.png">
<img width="1440" alt="OverviewRubberBandGuide3" src="https://user-images.githubusercontent.com/45598764/93534879-ba570900-f8fa-11ea-9414-a5efc4bb6000.png">

<img width="1428" alt="OverviewRubberBandMouseDraggingGuides" src="https://user-images.githubusercontent.com/45598764/93534898-c0e58080-f8fa-11ea-8958-98967122d971.png">
<img width="1431" alt="OverviewRubberBandMouseDraggingGuidesHf" src="https://user-images.githubusercontent.com/45598764/93534901-c4790780-f8fa-11ea-9a80-728c73fb8979.png">
<img width="1437" alt="OverviewRubberBandMouseDraggingGuides2" src="https://user-images.githubusercontent.com/45598764/93534869-b6c38200-f8fa-11ea-9400-5852f1fd881c.png">

**Comment by cmdcolin** [comment]
Can you use the format refName:coordinate (it is currently refName<space>coordinate)

Then I think it's probably ready for a PR:)

**Comment by cmdcolin** [comment]
Oh wait theres already a PR #1209

**Comment by teresam856** [comment]
Note: We may have to tackle long refNames in the future. Long refNames (e.g "ENSMUST00000167415.7|ENSMUSG00000086429.9|OTTMUSG00000034748.3|OTTMUST00000090750.1|Gt(ROSA)26Sor-204|Gt(ROSA)26Sor|551|antisense|") could cause problems in the tooltips. For now we will display the entire refNames.

**Comment by cmdcolin** [comment]
Addressed via #1209
